docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -1,246 +1,211 @@
 .. _real_and_lazy_data:
 
+
+.. testsetup:: *
+
+    import dask.array as da
+    import iris
+    import numpy as np
+
+
 ==================
 Real and Lazy Data
 ==================
 
-What is Real and Lazy Data?
----------------------------
+We have seen in the :doc:`user_guide_introduction` section of the user guide that
+Iris cubes contain data and metadata about a phenomenon. The data attribute of a
+cube contains the actual numerical values recorded for the phenomenon the cube describes.
+The data element of a cube is always an array, but the array may be either
+"real" or "lazy".
 
-Every Iris cube contains an n-dimensional data array, which could be real or
-lazy.
+In this section of the user guide we will look specifically at the concepts of
+real and lazy data as they apply to the cube and other data structures in Iris.
 
-Real data is contained in a NumPy array which has a shape, a data type, some
-other useful information and many data points, each of which use up a small
-allocation of memory.
 
-Lazy data is contained in a conceptual array which retains the information
-about its real counterpart but has no actual data points, so its memory
-allocation is much smaller.  This will be in the form of a Dask array.
-
-Arrays in Iris can be converted between their real and lazy states,
-although there are some limits to this process (*explain this better - what
-limits?*).
+What is real and lazy data?
+---------------------------
 
-The advantage of using lazy data is that it has a small memory footprint which
-enables the user to load and manipulate datasets that you would otherwise not
-be able to fit into memory.
+In Iris, we use the term **real data** to describe data arrays that are loaded
+into memory. Real data is typically provided as a
+`NumPy array <https://docs.scipy.org/doc/numpy/reference/generated/numpy.array.html>`_,
+which has a shape and data type that are used to describe the array's data points.
+Each data point takes up a small amount of memory, which means large NumPy arrays can
+take up a large amount of memory.
 
-However, in order to execute certain operations (such as calculations on actual
-data values) the real data must be realized.  Using Dask, the operation will
-be deferred until you request the result, at which point it will be executed
-using Dask's parallel processing schedulers.  The combination of these two
-behaviours can offer a significant performance boost.
+Conversely, we use the term **lazy data** to describe data that is not loaded into memory.
+(This is sometimes also referred to as **deferred data**.)
+In Iris, lazy data is provided as a
+`dask array <http://dask.pydata.org/en/latest/array-overview.html>`_.
+A dask array also has a shape and data type
```

Comment:
If you look at how NumPy refers to itself (i.e. in its docs), it is always in that form. I have a real bee in my bonnet that we should respect that in _all_ our docs where we refer to NumPy (especially as I'm quite hot that the name "Iris" is a Proper Noun, not an ACRONYM). 

Dask, meanwhile, is just a name and not even a proper name, though dask's docs seem to have a mix of dask capitalized and not capitalized. I've just gone for not capitalized as there doesn't seem to have a standardised name for dask...